### ISSUE TITLE
6.3.5- terraform apply error

### ISSUE THREAD

**Issue by samarthdd** [k8-proxy/test-installation-documentation#34]
![image](https://user-images.githubusercontent.com/64204445/104696760-c5993480-5734-11eb-89bd-c3e0b32485dc.png)

**Comment by Kirsten-Osborn** [comment]
@samarthdd there is a pre-req to create an azure-hosted dns zone - we've now added this to the pre-reqs listed in section 5 in v0.3 of the user installation guide.  The service principal requires collaborator access to the DNS zone.  I can see the error msg is referring to the Glasswall DNS.  Given the additional info, could you possibly re-test.

**Comment by samarthdd** [comment]
@Kirsten-Osborn @GINAGC 

 This is still not resolved for me.
 
I added DNS zone and DNS record using link  https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/add-users-azure-active-directory
 
Case 1.

Here DNS zone_name= prod.icap-proxy.curlywurly.me

Also icap-proxy.curlywurly.me is not there in DNS zone or If  icap-proxy.curlywurly.me DNS zone is in random resource :

Below error will appear:

![image](https://user-images.githubusercontent.com/64204445/105323759-85ccc400-5bf0-11eb-995e-51ac6531bac0.png)

Case 2.

Here DNS zone_name= prod.icap-proxy.curlywurly.me

Added icap-proxy.curlywurly.me to gw-icap-rg-dns resource.

Getting below error

![image](https://user-images.githubusercontent.com/64204445/105325343-7f3f4c00-5bf2-11eb-897d-5205e88f8e01.png)


Case 3. So i added this DNS zone  icap-proxy.curlywurly.me to gw-icap-rg-dns

Here DNS zone_name= prod.glasswall-prod-icap.com

got below error saying it is already exists.

Notice error  "on 218.248.112.1:53: no such host" in second line


Notice my dns zone in this case is prod.glasswall-prod-icap.com but from somewhere this icap-proxy.curlywurly.me is coming into picture. If it not added, will case 1  error will come, if added in DNS zone and gw-icap-rg-dns, getting case 3 error

![image](https://user-images.githubusercontent.com/64204445/105324303-418df380-5bf1-11eb-9abb-d9a832fd0788.png)

In conclusion:

Irrespective of dns_zone_name in main.tf "icap-proxy.curlywurly.me" is coming into picture from some code.

The terraform apply is creating prod.icap-proxy.curlywurly.me but throwing error no host found.

Still investigating this. But apart from DNS zone  name, looks like system needs more info about IP and other specs.

**Comment by samarthdd** [comment]
One of the things which I want to double check is if it dependent on 6.3.4
